src/ca65/expr.c

```diff
@@ -1099,6 +1099,15 @@ static ExprNode* Factor (void)
             N = Function (FuncAddrSize);
             break;
 
+	case TOK_ASIZE:
+            if (GetCPU() != CPU_65816) {
+                N = GenLiteralExpr (8);
+            } else {
+	        N = GenLiteralExpr (ExtBytes [AM65I_IMM_ACCU] * 8);
```

Comment:
I suspect that Oliver was thinking about two lines at the same time -- he got his lines crossed!  ;-)

His comment is for the third line above this one: `GetCPU ()`.

This line needs a space _removed_: `ExtBytes[AM65I_IMM_ACCU]`.